improver_tests/acceptance/test_apply_reliability_calibration.py

```diff
@@ -78,3 +78,18 @@ def test_no_calibration(tmp_path):
     args = [forecast_path, "--output", output_path]
     run_cli(args)
     acc.compare(output_path, forecast_path)
+
+
+def test_calibration_point_by_point(tmp_path):
+    """
+    Test point by point calibration of a forecast using a reliability
+    calibration table.
+    """
+    kgo_dir = acc.kgo_root() / "apply-reliability-calibration/basic"
+    kgo_path = kgo_dir / "kgo_point_by_point.nc"
+    forecast_path = kgo_dir / "forecast_point_by_point.nc"
```

Comment:
We might want to cut this input down even further, so that this test will be quicker to run.